src/index.html

```diff
@@ -12,50 +12,45 @@
   </head>
   <body>
     <main class="main">
-      <!-- <div class="container"> -->
       <form
       action="#"
-      method="POST"
+      method="GOT"
       data-qa="big"
-      class="container"
-      >
+      class="container">
         <input
-        type="text"
-        name="big"
-        id="search"
-        placeholder="Try &quot;Los Angeles&quot;"
-        class="box"
-        data-qa="keypress"
+          type="text"
+          name="big"
+          id="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="box"
+          data-qa="keypress"
         >
         <img
-        src="images/Search.svg"
-        alt="Search icon"
-        action="submit"
-        class="img">
+          src="images/Search.svg"
+          alt="Search icon"
+          action="submit"
+          class="img"
+        >
       </form>
-      <!-- </div> -->
-      <!-- <div class="container_small"> -->
-      <form
-      action="#"
-      method="POST"
+
+      <form action="#"
+      method="GOT"
```

Comment:
actually it's `GET`, but I think it's just missclick 😄

Reply:
Yes, it's a missclick. Thank you.